components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/handlers/throttling/ThrottleHandler.java

```diff
@@ -275,43 +275,42 @@ private boolean doRoleBasedAccessThrottlingWithCEP(MessageContext synCtx, Config
-                                        if (log.isDebugEnabled()) {
-                                            log.debug("Throttled with Condition :" + throttledCondition);
-                                        }
-                                        String combinedResourceLevelThrottleKey = resourceLevelThrottleKey + "_" +
-                                                throttledCondition;
+                                    String combinedResourceLevelThrottleKey = resourceLevelThrottleKey + "_" +
+                                            throttledCondition;
 
-                                        if (log.isDebugEnabled()) {
-                                            log.debug("Checking condition : " + combinedResourceLevelThrottleKey);
-                                        }
+                                    if (log.isDebugEnabled()) {
+                                        log.debug("Checking condition : " + combinedResourceLevelThrottleKey);
+                                    }
 
-                                        if (getThrottleDataHolder().isThrottled(combinedResourceLevelThrottleKey)) {
-                                            if (!apiLevelThrottledTriggered) {
-                                                isResourceLevelThrottled = isThrottled = true;
-                                            } else {
-                                                isApiLevelThrottled = isThrottled = true;
-                                            }
-                                            long timestamp = getThrottleDataHolder().
-                                                    getThrottleNextAccessTimestamp(combinedResourceLevelThrottleKey);
-                                            synCtx.setProperty(APIThrottleConstants.THROTTLED_NEXT_ACCESS_TIMESTAMP,
-                                                    timestamp);
+                                    if (getThrottleDataHolder().isThrottled(combinedResourceLevelThrottleKey)) {
+                                        if (!apiLevelThrottledTriggered) {
+                                            isResourceLevelThrottled = isThrottled = true;
+                                        } else {
+                                            isApiLevelThrottled = isThrottled = true;
                                         }
+                                        long timestamp = getThrottleDataHolder().
+                                                getThrottleNextAccessTimestamp(combinedResourceLevelThrottleKey);
+                                        synCtx.setProperty(APIThrottleConstants.THROTTLED_NEXT_ACCESS_TIMESTAMP,
+                                                timestamp);
                                     }
-                                } else {
+                                }
+                            } else {
+                                if (conditionGroupDTOs != null && conditionGroupDTOs.length > 0) {
+
```

Comment:
unnecessary new line?

Reply:
Fixed in https://github.com/wso2/carbon-apimgt/pull/9250